acceptance/ui/features/steps/generic_steps.rb

```diff
@@ -122,4 +135,38 @@ def sortColumn(category, sortOrder)
     while categoryLink[:class] != order do
         categoryLink.click()
     end
+end
+
+def removeAllEntries()
+    entries = page.all("[ng-repeat$='in $data']")
+    for i in 0..(entries.size - 1)
+        within(entries[i]) do
+            click_link_or_button("Delete")
+        end
+        click_link_or_button("Remove")
+    end
+end
+
+def getTableValue(valueOrTableUrl)
+    if valueOrTableUrl.start_with?("table://") == false
+        return valueOrTableUrl
+    end
+    parsedUrl = valueOrTableUrl.split(/\W+/)
```

Comment:
To be defensive here, please add a check for the length of parsedUrl. For now, a valid URL must have exactly 4 values - anything else is invalid.